.travis.yml

```diff
@@ -417,7 +417,17 @@ matrix:
       env:
         - SHARD="Python integration tests for pants - shard 7"
       script:
-        - ./build-support/bin/ci.sh -x -fkmsrjlpnt -i 6/7 "${SHARD}"
+        - ./build-support/bin/ci.sh -x -efkmsrjlpnt -i 6/7 "${SHARD}"
+
+    # Rust tests
+    - os: linux
+      dist: trusty
+      group: deprecated-2017Q2
+      sudo: required
+      language: python
+      python: "2.7.13"
+      script:
+        - ./build-support/bin/ci.sh -abcfjklmnprstx
```

Comment:
You need not emulate the SHARD env var style, but tacking on an ignored shard description positional are at the end here helps identify what the shard is testing when reading the Travis web ui.

Reply:
Done